documentation/usage/errorcodes/index.md

```diff
@@ -16,6 +16,38 @@ These error codes may appear from any command, and are indicative of more genera
 - **Caused by:** An unexpected error within Flyway (e.g. a null pointer exception)
 - **Solution:** Please contact support or create a [GitHub issue](https://github.com/flyway/flyway/issues)
 
+### `ERROR`
+- **Caused by:** An error due to invalid configuration or usage
```

Comment:
I think we should call out that the `Error` code is a catch all, for when we don't have a more specific code to produce. So if users are switching on error types, this is essentially the default case.

Reply:
see latest commit